NEWS.md

```diff
@@ -1,5 +1,17 @@
 ## 12.9.0 - Unreleased
 
+## 12.8.1 - Released (Quesnelia R1 2024)
+This release focused on to fix bugs relate to payment status, invoice and item records.
```

Comment:
I think should be `related to` instead of `relate to`

Reply:
Thanks, used your suggestion